Tests/Legacy/Shopware/Connect/Component/CategoryResolver/AutoCategoryResolverTest.php

```diff
@@ -78,33 +78,30 @@ public function testResolve()
         if (!$nikeBootsCategory) {
             $this->categoryResolver->convertNodeToEntity([
                 'name' => 'Boots',
-                'categoryId' => '/spanish/nike/boots'
+                'categoryId' => '/deutsch/nike/boots'
             ], $nikeCategory);
         }
 
         $categories = [
-            '/spanish/nike/tshirts' => 'Tshirts',
-            '/spanish/nike' => 'Nike',
-            '/spanish/adidas/boots' => 'Boots',
-            '/spanish/adidas' => 'Adidas',
-            '/spanish' => 'Spanish',
+            '/deutsch/nike/tshirts' => 'Tshirts',
+            '/deutsch/nike' => 'Nike',
+            '/deutsch/adidas/boots' => 'Boots',
+            '/deutsch/adidas' => 'Adidas',
+            '/deutsch' => 'Deutsch',
         ];
 
         $categoryModels = $this->categoryResolver->resolve($categories);
 
-        //Spanish category must not be created
-        $this->assertNull($this->categoryRepo->findOneByName('Spanish'));
         $this->assertEquals(
             $defaultCategoryId,
             Shopware()->Db()->fetchOne('SELECT parent FROM s_categories WHERE description = ?', ['Adidas'])
```

Comment:
Please ask Charly about that. In the past we just imported Adidas category into Deutsch

Reply:
in the past Adidas would get parent null and because of that it wasn't visible
it doesn't make sense to import it in the default category because the category names will be in Spanish and that would be pretty confusing